docker-compose.yml

```diff
@@ -14,11 +14,13 @@ services:
   # (1) Make sure you have raven-testdata locally (https://github.com/Ouranosinc/raven-testdata)
   # (2) Set the left part (local path) in the `volumes` section below to the right path
   # (3) Run: docker-compose run tests
+  #
+  # In case of weird error, rebuild from scratch: docker-compose build --no-cache tests
```

Comment:
This does not run the tests on my end. I needed to do `docker-compose run tests` after the build.

Reply:
Yes the command is just to rebuild.  Some context, the rebuild is done automatically by `docker-compose run tests` only if the image do not exist.  But if the image already exist from a previous run, it should still be fine since we change dependencies rarely and we volume mount in the raven repo so the code is always fresh.  But weird error will happen when the dependencies are out-of-date (ex: xclim too old).

Reply:
oui en fait je le dis à l'étape (3) juste en haut, mais j'avoue que c'est un peu boiteux mon affaire ..

Reply:
Pushed new commit with some clarification a385439